### ISSUE TITLE
Doesn't work with the latest Anki 2.1.15 

### ISSUE THREAD

**Issue by OnkelTem** [Stvad/CrowdAnki#70]
Hi Vladyslav!

Just installed 2.1.15 (Linux) and it cannot load CrowdAnki. The version was grabbed from here:
https://ankiweb.net/shared/info/1788670778

The error message is displayed upon Anki for Desktop start:
```
An add-on you installed failed to load. If problems persist, please go to the Tools>Add-ons menu, and disable or delete the add-on.

When loading 'CrowdAnki JSON exportimport Edit history Collaborate on deck creation':
Traceback (most recent call last):
  File "aqt/addons.py", line 80, in loadAddons
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/__init__.py", line 5, in <module>
    from . import main
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/main.py", line 8, in <module>
    from .anki.hook_vendor import HookVendor
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/anki/hook_vendor.py", line 5, in <module>
    from ..export.anki_exporter_wrapper import exporters_hook
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/export/anki_exporter_wrapper.py", line 3, in <module>
    from .anki_exporter import AnkiJsonExporter
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/export/anki_exporter.py", line 10, in <module>
    from ..representation import deck_initializer
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/representation/deck_initializer.py", line 1, in <module>
    from functional import seq
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/__init__.py", line 7, in <module>
    from functional.streams import seq, pseq
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/streams.py", line 11, in <module>
    from functional.execution import ExecutionEngine, ParallelExecutionEngine
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/execution.py", line 2, in <module>
    from functional.util import compose, parallelize
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/util.py", line 7, in <module>
    from multiprocessing import Pool, cpu_count
ModuleNotFoundError: No module named 'multiprocessing'
```

Then I tried the latest dev version of CrowdAnki but it didn't work either:

```
An add-on you installed failed to load. If problems persist, please go to the Tools>Add-ons menu, and disable or delete the add-on.

When loading '1788670778':
Traceback (most recent call last):
  File "aqt/addons.py", line 80, in loadAddons
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/__init__.py", line 5, in <module>
    from . import main
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/main.py", line 8, in <module>
    from .anki.hook_vendor import HookVendor
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/anki/hook_vendor.py", line 6, in <module>
    from ..export.anki_exporter_wrapper import exporters_hook
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/export/anki_exporter_wrapper.py", line 3, in <module>
    from .anki_exporter import AnkiJsonExporter
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/export/anki_exporter.py", line 11, in <module>
    from ..representation import deck_initializer
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/representation/deck_initializer.py", line 1, in <module>
    from functional import seq
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/__init__.py", line 7, in <module>
    from functional.streams import seq, pseq
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/streams.py", line 11, in <module>
    from functional.execution import ExecutionEngine, ParallelExecutionEngine
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/execution.py", line 2, in <module>
    from functional.util import compose, parallelize
  File "/home/fantomas/.local/share/Anki2/addons21/1788670778/dist/functional/util.py", line 7, in <module>
    from multiprocessing import Pool, cpu_count
ModuleNotFoundError: No module named 'multiprocessing'
```

**Comment by ohare93** [comment]
Looks like the exact issue I had when installing Anki on Linux :thinking: See here: https://github.com/Stvad/CrowdAnki/issues/62#issuecomment-529150013 

> Hey, that fixed it! Installing with `sudo apt-get install anki` then a quick `sudo apt-get install python3-distutils` fixed my issues. Thank you :+1:  Now I can actually program :sweat_smile: 

Seemed to only be an issue with the downloaded version of Anki, but installing it this way fixed my issue :+1:

**Comment by OnkelTem** [comment]
Nah, unfortunately it hasn't resolved the issue :(
I installed `python3-distutils` using apt, but not `anki` because it's too old there while I wanted to play exactly 2.1.15. 
At the same time I have no any problems with 2.1.13.

**Comment by OnkelTem** [comment]
summoning @Stvad  :)

**Comment by Stvad** [comment]
🧞‍♂️

**Comment by Stvad** [comment]
@OnkelTem @ohare93 is right in general (I'm not sure how `python3-distutils` related to this one though)
 
The "official" Anki packaging is broken for Linux (by design 🤦‍♂️) - it cuts out parts of python standard library. The fix is to use the version installed using the system package manager (it'd cause Anki to use system python). 

Regarding getting latest versions of Anki - not sure, maybe there is a PPA or something that'd feature them?

**Comment by OnkelTem** [comment]
Erm.. So you guys mean installing anki from the repo but not running it, but instead - run local version of Anki for instance - 2.1.15? %-) I can try this out

**Comment by Stvad** [comment]
No, I meant actually using Anki from the repo and finding a repo with a newer version of Anki if need be ;)

You can probably make the Anki version from the Anki website work with system python, but I don't have instructions for doing that

**Comment by icepie** [comment]
> Nah, unfortunately it hasn't resolved the issue :(
> I installed `python3-distutils` using apt, but not `anki` because it's too old there while I wanted to play exactly 2.1.15.
> At the same time I have no any problems with 2.1.13.

Hey guys! I meet the same wrong with the 2.1.15 in Linux, but I fix it when I update anki to the 2.1.120 :)
